README.md

```diff
@@ -1 +1,20 @@
 # hw01
+
+
+# About Me
+My name is **Anastasiya Apalkova** and I am a student in the MAPSS program concentrating in psychology. 
+I was born in Ukraine and moved to Chicago with my family when I was seven years old. 
+***
+![Ukraine Map](https://res.cloudinary.com/fen-learning/image/upload/c_limit,w_505,h_490/infopls_images/images/mukraine.gif) ![picture of me](pic4.jpg)
```

Comment:
Nice idea to store the image in the repo for embedding. Easy approach if the image is not already hosted online